Sources/SwiftFormat/Core/Pipelines+Generated.swift

```diff
@@ -76,17 +76,16 @@ class LintPipeline: SyntaxVisitor {
     visitIfEnabled(AllPublicDeclarationsHaveDocumentation.visit, for: node)
     visitIfEnabled(AlwaysUseLowerCamelCase.visit, for: node)
     visitIfEnabled(BeginDocumentationCommentWithOneLineSummary.visit, for: node)
-    visitIfEnabled(DontRepeatTypeInStaticProperties.visit, for: node)
     visitIfEnabled(NoLeadingUnderscores.visit, for: node)
     visitIfEnabled(TypeNamesShouldBeCapitalized.visit, for: node)
     visitIfEnabled(UseTripleSlashForDocumentationComments.visit, for: node)
     return .visitChildren
   }
   override func visitPost(_ node: ClassDeclSyntax) {
     onVisitPost(rule: AllPublicDeclarationsHaveDocumentation.self, for: node)
+    visitPost(AlwaysUseLowerCamelCase.visitPost, for: node)
```

Comment:
Is this an unrelated change? And should this be `onVisitPost`. I don’t actually know but `visitPost` sticks out to me here.

Reply:
My original implementation used `visitPost`, but there was a problem because the generated pipeline doesn't pass on `visitPost` calls to rules that override those methods. `onVisitPost` only does some cleanup of the pipeline. So the code would work as a standalone visitor, but not as part of the pipeline. I ended up changing the implementation to not use `visitPost`, but I did keep the fix to the code generator in.

Reply:
OK, I’m not too familiar with how the entire pipeline works but I trust you on this 😉

Reply:
Can we drop this and the code generator changes then, if they're not being used?

Reply:
@allevato I could split it out into another change, but I still want to put it in because AFAICT it's a bug in the pipeline generator (`visitPost` is never called) and I have other changes in progress that depend on that visitor method being called. Should I go ahead and split this into two separate changes?